### ISSUE TITLE
Divide By Zero Error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#114]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?

Normal running of gphotoframe.


What version of the product are you using? On what operating system?

2.0-a2
Fedora 18


Please provide any additional information below.

```

Original issue reported on code.google.com by `lilzebr...@gmail.com` on 4 Jul 2014 at 4:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `lilzebr...@gmail.com` on 4 Jul 2014 at 4:21

Attachments:
- [gphotoframe-ABRT screen 2014-07-04 11:11:08.png](https://storage.googleapis.com/google-code-attachments/gphotoframe/issue-114/comment-1/gphotoframe-ABRT screen 2014-07-04 11:11:08.png)

**Comment by GoogleCodeExporter** [comment]
```
Could you show me the image file that couses the error?
```

Original comment by `yendo0206` on 12 Aug 2014 at 2:19

**Comment by GoogleCodeExporter** [comment]
```
Fixed in revision d8b5947490f4.
```

Original comment by `yendo0206` on 17 Nov 2014 at 4:56
- Changed state: **Fixed**
